Documentation/Experimental/SceneUnderstanding/README_SceneUnderstanding.md

```diff
@@ -0,0 +1,99 @@
+# MRTK Scene Understanding Demo
+
+This is an __experminental__ prelease that integrates [Scene Understanding](https://docs.microsoft.com/en-us/windows/mixed-reality/scene-understanding) (SU) as a MRTK.
+
+SU - in a nutshell, will return a semantic representation of scene entities as well as their geometric forms on __HoloLens 2__.
+
+## Goals and features of the demo
+* Demo scene with visualization of Scene Objects with options for configuring the observer
+* Example script that excersises the event system demonstrating common tasks
+* Save device room data for offline development
+* In-editor workflow for developing with previously saved room data (.bytes files)
+* For the first time, access to the highest possible resolution of the world mesh!
+
+## Observer overview
+
+When asked, the SU observer will return Scene Objects with attributes useful for mixed world development. What is returned is dependent on the configuration of the observer when fetched. For instance, if you want the occlusion mask, the observer must be configured to generate quads.
+
+It is expected the majority of "real" applications will never show the underlying data in raw form.
+
+## Background and development notes
+
+These links cover the underlying technology the Spatial Awareness service is using behind the scenes.
+
+https://docs.microsoft.com/en-us/windows/mixed-reality/scene-understanding
+
+https://docs.microsoft.com/en-us/windows/mixed-reality/scene-understanding-sdk
+
+The Scene Understanding observer inherits from older spatial awareness concepts so the implementation is and odd fit in places.
+
+Eventually this system should replace the existing SpatialAwareness observer.
```

Comment:
This seems like a dangerous thing to say? @davidkline-ms 

especially since this is designed for HL2 and not other platforms as well as does not elaborate on how, when or in what what this will replace the existing system